includes/admin_import.php

```diff
@@ -4,743 +4,1212 @@
+			padding:10px 20px 0px 0px;
+			line-height: 50%;
+			height: auto;
+		}
+
+		#import_settings_wrap .li .p {
+			margin: 0;
+			padding: 0;
+		}
+
+		#import_settings_wrap .import-form {
+			padding:10px 20px 0px 0px;
+			line-height: 50%;
+			height: 325px;
+		}
+
+		#import-1 .li .p {
+			margin: 0;
+			padding: 0;
+		}
+
+		#ul_horiz_summary li {
+		display: inline;
+		}
+
+		</style>
+
+		<!-- Admin page content should all be inside .wrap -->
+		<div id="import_settings_wrap" class="wrap">
+
+			<?php if (!is_ssl()) { ?>
+				<div class="notice notice-warning inline">
+					<p><?php _e('If you enable SSL (https), your users will be able to search near their location.', '12-step-meeting-list') ?></p>
+				</div>
+			<?php } ?>
+
+			<div id="poststuff" >
+				<div id="post-body" class="inside ">
+					<div id="post-body-content" >
+
+						<?php if ($error) { ?>
+							<div class="error inline">
+								<p><?php echo $error ?></p>
+							</div>
+						<?php } elseif ($total = count($meetings)) { ?>
+							<div id="tsml_import_progress" class="progress" data-total="<?php echo $total ?>">
+								<div class="progress-bar"></div>
+							</div>
+							<ol id="tsml_import_errors" class="error inline hidden"></ol>
+						<?php } ?>
+
+						<?php if (empty($tsml_mapbox_key) && empty($tsml_google_maps_key)) { ?>
+							<div class="notice notice-warning inline" style="margin: 1px 0 22px;">
+								<div class="inside" >
+									<h1>Enable Maps on Your Site</h1>
+									<p>If you want to enable maps on your site you have two options: <strong>Mapbox</strong> or <strong>Google</strong>. *See the Google restriction noted below.
+										They are both good options, although Google is not completely supported by all our features! In all likelihood neither one will charge you money. Mapbox gives
+										<a href="https://www.mapbox.com/pricing/" target="_blank">50,000 free map views</a> / month, Google gives
+										<a href="https://cloud.google.com/maps-platform/pricing/" target="_blank">28,500 free views</a>.
+										That's a lot of traffic!
+									</p>
+									<p>To sign up for Mapbox <a href="https://www.mapbox.com/signup/" target="_blank">go here</a>. You will only need
+										a valid email address, no credit card required. Copy your access token and paste it below:</p>
+									<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px 0;">
+										<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+										<div class="input">
+											<input type="text" name="tsml_add_mapbox_key" placeholder="Enter Mapbox access token here">
+										</div>
+										<div class="btn">
+											<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+										</div>
+									</form>
+
+									<p>*Please note: Only Mapbox is supported by our <b>TSML UI</b> user interface!  Google is not an option.
```

Comment:
Might be cleaner to write this: "Please note: TSML UI only supports Mapbox, not Google."